Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -952,7 +968,7 @@ internal enum EventTriggering {
                     return this.Caption;
 
                 case __VSHPROPID.VSHPROPID_ProjectDir:
-                    return this.ProjectFolder;
+                    return this.ProjectHome;
```

Comment:
I wasn't sure about this, msdn just says this is a path to project directory. We had few components that assumed this would return ProjectHome. I've changed them to use ProjectHome from EnvDTE.Properties instead. So I'm reverting this back to ProjectFolder.